css/styles.css

```diff
@@ -0,0 +1,56 @@
+body {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto",
+    "Helvetica Neue", Arial, sans-serif;
+}
+
+#main {
+  background-color: #fff;
+  padding: 30px;
+}
+
+#img-div {
+}
+
+.container {
+  text-align: center;
+  background-color: #eee;
+  padding: 15px 0;
+}
+.container_a {
+  background-color: #fff;
+}
+.container_a img {
+  display: block;
+  margin: 0 auto;
+}
+#img_description {
+  text-align: center;
+  padding: 15px 30px;
+}
+
```

Comment:
Please resize your image container as you observe it overflows beyond the x-axis.

Please see to it that the entire body is given an overflow-x to be hidden.